docs/design/flow-loop/flow-loop.md

```diff
@@ -0,0 +1,44 @@
+# Flow loop feature
+
+## Overview
+
+Flow loop feature designed for flow path testing. Loop provides additional flow rules on one of the terminating switch so any flow traffic is returned to switch-port where it was received. 
+Such flow has `looped=true` flag and support all flow operations. When loop removed system should restore original flow rules.
```

Comment:
where you going to store this flag from db perspective, please describe

Reply:
It's described in Details section.

Reply:
no details, which attributes will this looped on should include?

Reply:
Reworked from relation to additional properties to flow node.